omnibus/config/projects/agent.rb

```diff
@@ -230,6 +230,12 @@
     include_apminject = "true"
   end
 
+  include_procmon = "false"
+  if not windows_arch_i386? and ENV['WINDOWS_DDPROCMON_DRIVER'] and not ENV['WINDOWS_DDPROCMON_DRIVER'].empty?
```

Comment:
Couldn't you check if the file exists rather than probing the environment? If I understand your change correctly, `security-agent.exe` will only be there if it was built, based on the same condition in `datadog-agent.rb`
This would be simpler to read and to toggle IMO